run/fvde2john.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+# Usage: python3 fvde2john.py <image_file>
+# The partition table is parsed to find the boot volume, often named 'Recovery HD'. The boot volume can be identified by its type GUID: 426F6F74-0000-11AA-AA11-00306543ECAC.
+# The boot volume contains a file called `EncryptedRoot.plist.wipekey`. This is stored on the volume at `/com.apple.boot.X/System/Library/Caches/com.apple.corestorage/EncryptedRoot.plist.wipekey`, where `X` is variable but is often `P` or `R`. This plist file is encrypted with AES-XTS; the key is found in the CoreStorage volume header, and the tweak is b'\x00' * 16.
+# The decrypted plist contains information relating to the user(s). This includes the salt, kek and iterations required to construct the hash as well as information such as username and password hints (if present).
+
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+import plistlib
+import os
+import argparse
+import pytsk3    # pip install pytsk3
```

Comment:
For third party dependencies, we usually add a try-except block in order to guide the user towards the installation of the correct package.
I also moved the `cryptography` import below the stdlib imports in my suggestion.

```suggestion
import argparse
import os
import plistlib
import sys

try:
    from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
except ImportError:
    print("cryptography is missing, run 'pip install --user cryptography' to install it!")
    sys.exit(1)
try:
    import pytsk3
except ImportError:
    print("pytsk3 is missing, run 'pip install --user pytsk3' to install it!")
    sys.exit(1)
```